### PR TITLE
typescript/two-fer: add mentoring notes

### PR DESCRIPTION
**Pull request by SleeplessByte**
(I'm the only member of @exercism/website-copy-typescript, so someone please approve).

### CODE REVIEW

**Comment by SleeplessByte** [comment]
note to self, copy changes by @tejasbubane on #1086

File: tracks/typescript/exercises/two-fer/mentoring.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  I'm approving your current submission so you can continue down the track, but
+  feel free to iterate, improve and submit another solution. I will continue to
+  mentor your future iterations.
+
+### Common suggestions
+
+This is the first mentored exercise, so you may use this to determine a
+student's skill, help them set up their test suite, and generally point out
+certain things. In the JavaScript track, this exercise is not meant to be a
+barrier so in general, don't be harsch but set the tone for the rest of the
```
Comment (F3PiX) [inline tracks/typescript/exercises/two-fer/mentoring.md]:
```suggestion
barrier so in general, don't be harsh but set the tone for the rest of the
```

File: tracks/typescript/exercises/two-fer/mentoring.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+}
+```
+
+#### Approvability
+
+You should **_only approve_** this if it matches the solutions above,
+with a few exceptions where you may be lenient:
+
+- a student uses CommonJS style `module.exports`
+- a student does omitted the return type `: string`
```
Comment (F3PiX) [inline tracks/typescript/exercises/two-fer/mentoring.md]:
```suggestion
- a student omitted the return type `: string`
```

**Comment by SleeplessByte** [comment]
Thank you @F3PiX :heart:

**Pull request merged**